agent/containers/images/visualizers/README.md

```diff
@@ -7,6 +7,9 @@
 - Directions:
   -   `podman pull pbench/live-metric-visualizer`
   -   `podman run --network host pbench/live-metric-visualizer`
+  -   To visualize a run on a remote host, add `-e HOST=<host>`
```

Comment:
The visualizer will only ever need to know the host where the TDS is running, as I'm assuming prometheus/pmproxy will always be on that host. Is this ever not the case?

Reply:
Yes, with containers, once can run a Redis server on host and the Tool Data Sink could be on another host.

Reply:
Does this mean that pmproxy or prometheus are ever run on the redis host rather than on the TDS host?

Reply:
I think we should probably focus on two environments:

1. Pbench orchestrates everything; we know exactly where every piece is connected because we put it there, and
2. Pbench is controlled through the APIs, with something external orchestrating... we know where nothing is unless we're told explicitly

The "default" self-orchestrated configuration is easy to manage if we support generalized external orchestration, because we just provide our defaults for every option. But anything that's hard coded may interfere with external orchestration scenarios.

Reply:
If a visualizer requires some sub-system for its operation, we either need to provide it explicitly internal to the container and ensure it is available and running in the container, or we need to have that sub-system provided to us explicitly.

Since we require a Redis instance for the operation of the other parts of the service, we can use the instance provided externally.  If one is not provided externally, then we need to ensure a local one is used.

See the comments in the `DockerfilePostPCP` and `run.py` ...